examples/ez-food/order-svc/tests/service_test.bal

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/http;
+import ballerina/io;
+import ballerina/test;
+
+http:Client testClient = check new ("http://localhost:9090");
+
+// Before Suite Function
+
+@test:BeforeSuite
+function beforeSuiteFunc() {
+    io:println("I'm the before suite function!");
+}
```

Comment:
IMO we dont need this empty BeforeSuite function

Reply:
Fixed with [65d6dfae0638e22f233d2fc383a47211cf5f23ea](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/119/commits/65d6dfae0638e22f233d2fc383a47211cf5f23ea)